.pubnub.yml

```diff
@@ -1,8 +1,13 @@
 name: unreal-engine
-version: 0.1.0
+version: 0.2.3
 schema: 1
 scm: github.com/pubnub/unreal-engine
 changelog: 
+  - date: 2024-11-28
+    version: 0.2.3
```

Comment:
Judging from `pattern` to replace version in `.pubnub.yml` (and already existing tags) there is expected tag prefixed with `v`, so each `version` entry in `changeling` should be prefixed with `v` as well.